scene/gui/text_edit.cpp

```diff
@@ -923,6 +924,26 @@ void TextEdit::_notification(int p_what) {
 						}
 					}
 
+					int indent_level = get_indent_level(i);
+					if (draw_indent_guides && indent_level > 0) {
+#ifdef TOOLS_ENABLED
+						int indent_size = EditorSettings::get_singleton()->get("text_editor/indent/size");
```

Comment:
Just saw this one, but `indent_size` is already defined in the header [here.](https://github.com/godotengine/godot/blob/master/scene/gui/text_edit.h#L265) No need to go to `EditorSettings`